### PR TITLE
Freshdesk widget

### PR DESCRIPTION
**Pull request by ankero**
Add freshdesk widget as an example

### CODE REVIEW

File: examples/FreshdeskWidget/package.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{
+  "name": "happeofreshdeskwidget",
```
Comment (mikamaunula) [inline examples/FreshdeskWidget/package.json:2]: Please rename to follow common npm package naming e.g. `happeo-freshdesk-widget`

File: examples/FreshdeskWidget/src/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      this,
+    );
+  }
+}
+const slug = "freshdesk-help-kk12b18hooxihmxbad15";
+
+window.customElements.get(slug) ||
+  window.customElements.define(slug, happeoCustomReactWidget);
+
+!(function () {
```
Comment (mikamaunula) [inline examples/FreshdeskWidget/src/index.js:19]: What does this do?

**Review by mikamaunula — APPROVED** [review]

**Review by mikamaunula — APPROVED** [review]

**Comment by loicmasson2** [comment]
@mikamaunula You can change it to this slug
`testfreshdesk-f03zjl9klwwkmtf65dns`

And on staging we have a custom app called TestFreshdesk

**Comment by mikamaunula** [comment]
> @mikamaunula You can change it to this slug `testfreshdesk-f03zjl9klwwkmtf65dns`
> 
> And on staging we have a custom app called TestFreshdesk

Seems that the browser crashes with the wrong widget id. Crash can be fixed with catching the error

**Pull request merged**